### DISCUSSION TITLE
Completly Broken Display

### DISCUSSION THREAD

**Discussion by Tularis1** [valinet/ExplorerPatcher#830] [Q&A] [answered]
So after a reboot my screen now looks like this..

![image](https://user-images.githubusercontent.com/48767622/153161734-32165205-b631-4914-af1b-3ef508bfb6b0.png)

Win11 Pro Ver 21H2 (22000.493)

Any anyone know of a fix?

I love this software so much I don't want to have to go back to grouped Windows :(

**Comment by Tularis1** [comment, marked as answer]
Nevermind. I didn't see the Restore Default Settings. Which has fixed the issue.

Which is odd bacause I didnt set any. oh well :)

**Comment by upwut** [comment]
Mine is doing the same thing. Working off two monitors, but it's only happening on the primary. Restore default doesn't fix it.
-Win11 Pro Ver 10.0.22000 Build 22000
-ExplorerPatch Ver 22000.469.41.16
-No 3rd party tweaks
![image](https://user-images.githubusercontent.com/32403478/153232966-f7141240-0a4d-408b-b63b-6fec213eddf4.png)

**Reply by valinet** [reply]
Doing what?

**Reply by upwut** [reply]
The taskbar is double the height it should be for no reason.

**Reply by valinet** [reply]
Yeah, make sure it’s not locked (right click it, uncheck Lock taskbar) and then drag its edge down.

**Reply by upwut** [reply]
So simple. Thanks!!

**Reply by Tularis1** [reply]
Do you mean that yours is broken because it double hight?
I set mine to be double hight on purpose.

My issue was that objects were not being rendered correctly.
